src/utils/CharactersPageUtils.js

```diff
@@ -101,3 +106,16 @@ export const prepareEditCharacterData = ({ formData, character }) => {
     birthYear,
   }
 }
+
+export const prepareSpeciesOptions = ({ speciesOptions }) => {
```

Comment:
Make sure you are only passing the data the function needs to run an nothing more. This function should only accept an array of objects